src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -364,103 +454,131 @@ public void ShadowCopyFiles()
         [Fact]
         public void AppendPrivatePath()
         {
-            AppDomain.CurrentDomain.AppendPrivatePath("test");
+            RemoteInvoke(() => {
+                AppDomain.CurrentDomain.AppendPrivatePath("test");
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void ClearPrivatePath()
         {
-            AppDomain.CurrentDomain.ClearPrivatePath();
+            RemoteInvoke(() =>
+            {
+                AppDomain.CurrentDomain.ClearPrivatePath();
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void ClearShadowCopyPath()
         {
-            AppDomain.CurrentDomain.ClearShadowCopyPath();
+            RemoteInvoke(() => {
+                AppDomain.CurrentDomain.ClearShadowCopyPath();
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void SetCachePath()
         {
-            AppDomain.CurrentDomain.SetCachePath("test");
+            RemoteInvoke(() => {
+                AppDomain.CurrentDomain.SetCachePath("test");
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void SetShadowCopyFiles()
         {
-            AppDomain.CurrentDomain.SetShadowCopyFiles();
+            RemoteInvoke(() => {
+                AppDomain.CurrentDomain.SetShadowCopyFiles();
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void SetShadowCopyPath()
         {
-            AppDomain.CurrentDomain.SetShadowCopyPath("test");
+            RemoteInvoke(() => {
+                AppDomain.CurrentDomain.SetShadowCopyPath("test");
+                return SuccessExitCode;
+            }).Dispose();
         }
-#pragma warning restore 618
 
+#pragma warning restore 618
         [Fact]
         [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Does not support Assembly.LoadFile")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
I'll leave this as is since I'm not disabling it and just leaving unchanged - there is some difference in behavior here which I don't want to investigate, since this has no reason it is really easy to grep. I'm hoping we will do some large sweep of all Skipped tests without a reason so that owners can help with understanding some differences